lib/card.rb

```diff
@@ -0,0 +1,9 @@
+class Card
+  attr_accessor :answer, :category
```

Comment:
Curious why you might use an attr_accessor for these two properties specifically, and not the `:question` as well?